config/suppressions.xml

```diff
@@ -15,4 +15,6 @@
             files="GeneratePatchFileLauncher|GeneratePatchFileWithGitCommandLauncher\.java"/>
     <suppress checks="UncommentedMain"
             files="GeneratePatchFileLauncher|GeneratePatchFileWithGitCommandLauncher\.java"/>
+    <suppress checks="AvoidDoubleBraceInitialization"
+              files="JavaPatchFilterElement\.java"/>
```

Comment:
Do not suppress checkstyle violations. Fix it

Reply:
done